.github/workflows/ci.yml

```diff
@@ -12,21 +12,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - name: Fetch path to Zcash parameters
-        working-directory: ./zcash_proofs
-        shell: bash
-        run: echo "ZCASH_PARAMS=$(cargo run --release --example get-params-path --features directories)" >> $GITHUB_ENV
```

Comment:
We can delete the `get-params-path` example now; it was only added for use in our CI. (However, we should verify that zebrad's CI isn't also using it; I think they might have been at one stage.)

Reply:
`git grep` shows no usage of `get-params-path` in the zebrad repo.

Reply:
Aha, what they actually did is copied those files into their CI. They removed their copies in https://github.com/ZcashFoundation/zebra/pull/7800.

Reply:
So yes, we can just remove `zcash_proofs/examples/` entirely (I don't think we need to offer Sprout parameter downloading examples either).

Reply:
Removed.